web-components/src/components/table/Table.ts

```diff
@@ -0,0 +1,128 @@
+import reset from "@/wc_scss/reset.scss";
```

Comment:
please add the same disclaimer we have in other files

Reply:
Yes, it's just draft PR